app/models/accounting/quickbooks/updater.rb

```diff
@@ -111,6 +113,16 @@ def extract_qb_data(txn)
         txn.save!
       end
 
+      def lookup_currency(txn)
+        project = txn.project
```

Comment:
This variable seems unnecessary to me

Reply:
I just felt weird calling `txn.project` in two different places